index.bs

```diff
@@ -435,8 +435,8 @@ interface ControlledFrame : HTMLElement {
 
     // Scripting methods.
     undefined addContentScripts(sequence<ContentScriptDetails> contentScriptList);
-    Promise<any> executeScript(optional InjectDetails details);
-    Promise<undefined> insertCSS(optional InjectDetails details);
+    Promise<any> executeScript(optional InjectDetails details = {});
+    Promise<undefined> insertCSS(optional InjectDetails details = {});
```

Comment:
This appears to say that executeScript() and insertCSS() could be called with {} or without any args. Can you help me understand its intent?

Reply:
Copying my reply that I did through email here for context:

Ideally I wouldn't want this to be defined like this, but the WebIDL
checker fails because InjectDetails doesn't have any required fields.

If we set both fields in InjectDetails to required, then that also doesn't
work well with the stipulation that only one of them should be defined.

So what this tells me is that InjectDetails needs to be modified to avoid
this.

Reply:
I understand. Please add an item to our backlog to track resolving this issue. Thanks!